docs/source/workflows/design_workflows.rst

```diff
@@ -132,9 +127,6 @@ You can to look up what :doc:`score <scores>` was used for a particular executio
     score = execution.score
     descriptors = execution.descriptors
 
-.. warning::
-    
-    Executing DesignWorkflows using `workflow.executions` is deprecated, please use `workflow.design_executions` as described above instead.
```

Comment:
same here, the example below this line is now a repeat of the one above

Reply:
also, the stuff describing how candidates used to be a dictionary can be removed